velox/expression/tests/CastExprTest.cpp

```diff
@@ -798,6 +797,15 @@ TEST_F(CastExprTest, primitiveValidCornerCases) {
   setCastIntByTruncate(true);
   // To integer.
   {
+    // Valid strings.
+    testCast<std::string, int8_t>("tinyint", {"1.2"}, {1}, false);
```

Comment:
can you also add a cases with a larger fractional part? like 1.23444 or ".2355", etc